Cart Flow/Custom Views/Utilities/SceneDelegate.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  SceneDelegate.swift
+//  Cart Flow
+//
+//  Created by Scott Quintana on 8/3/20.
+//  Copyright © 2020 Scott Quintana. All rights reserved.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
+    
+    var navigationController: UINavigationController?
+
+
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+       
+        guard let windowScene = (scene as? UIWindowScene) else { return }
+        
+        window = UIWindow(frame: windowScene.coordinateSpace.bounds)
+        window?.windowScene = windowScene
+        window?.rootViewController = UINavigationController(rootViewController: ListSelectVC())
+        configureNavigationBar()
+        window?.makeKeyAndVisible()
+    }
+
+
+    func configureNavigationBar() {
+        UINavigationBar.appearance().tintColor = .black
+        UINavigationBar.appearance().barTintColor = .white
+        UINavigationBar.appearance().backgroundColor = .white
+        UINavigationBar.appearance().titleTextAttributes = [NSAttributedString.Key.foregroundColor : UIColor.black]
```

Comment:
Make sure to test you app in light and dark mode. You're probably fine, but hardcoding these `.black` and `.white` values may look weird in either light or dark mode.

Reply:
I initially had more support for dark mode in the app, and have since gone back to not wanting to support it because I didn't like how it was working with the overall UI. Will see if I can think of a way to make it look how I want and support both modes.

Reply:
You can also opt out of light and dark modes by setting a preferred mode for your app. See this StackOverflow answer: https://stackoverflow.com/questions/56537855/is-it-possible-to-opt-out-of-dark-mode-on-ios-13